src/back8sup.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# -*- coding: utf-8 -*-
+#
+# back8sup - backup your k8s api resources easily on a PV of your choice
+#
+# Copyright (C) 2020 Adfinis AG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# SPDX-FileCopyrightText: 2020 Adfinis AG
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# Environment variables for configuration
+
+API_ENDPOINT=${API_ENDPOINT:-https://kubernetes.local:6443}
+CA_CERT=${CA_CERT:-/etc/ssl/ca.crt}
+TOKEN_FILE=${TOKEN_FILE:-/var/run/secrets/sa}
+DST_FOLDER=${DST_FOLDER:-/mnt/back8sup}
+CONFIGMAP_PATH=${CONFIGMAP_PATH:-/etc/config.yaml}
+EXPORT_FORMAT=${EXPORT_FORMAT:-yaml}
```

Comment:
would readonly make sense for these?
```suggestion
readonly API_ENDPOINT=${API_ENDPOINT:-https://kubernetes.local:6443}
readonly CA_CERT=${CA_CERT:-/etc/ssl/ca.crt}
readonly TOKEN_FILE=${TOKEN_FILE:-/var/run/secrets/sa}
readonly DST_FOLDER=${DST_FOLDER:-/mnt/back8sup}
readonly CONFIGMAP_PATH=${CONFIGMAP_PATH:-/etc/config.yaml}
readonly EXPORT_FORMAT=${EXPORT_FORMAT:-yaml}
```